packages/connect-node-test/src/crosstest/ping_pong.spec.ts

```diff
@@ -27,9 +27,18 @@ describe("ping_pong", () => {
     [
       // This test does full duplex, receiving before it closes the input.
       // In this case, your connect fetch transport never issues the request.
-      "connect fetch transport (binary) against Node.js (http)",
-      "connect fetch transport (JSON) against Node.js (http)",
-      "gRPC-web fetch transport (binary) against Node.js (http)",
+      "@bufbuild/connect-web (Connect, binary) against connect-go (h1)",
+      "@bufbuild/connect-web (Connect, binary) against connect-go (h2)",
+      "@bufbuild/connect-web (Connect, binary) against @bufbuild/connect-node (h1)",
```

Comment:
The list of transports to test is growing. Renamed the keys for better readability.